src/app/campaign/flight/flight.container.ts

```diff
@@ -43,6 +44,7 @@ import { map } from 'rxjs/operators';
 })
 export class FlightContainerComponent implements OnInit, OnDestroy {
   private currentFlightId: string;
+  private dailyMinimum: number;
```

Comment:
I'm not sure I like this dailyMinimum stored here on the container component. It's set from the goal-form and used for requesting the availability preview when the date, series, or zone fields change and the availability preview needs to be reloaded. I could probably store it in the campaign state... 🤔

Reply:
Yeah, I'd agree that a non-static value used to make API requests should probably live in state somewhere.

Reply:
I made an attempt at this, but the currentFlightId not being on the state added complications. I had to store a dailyMinimum per flight. Then I tried to set the Observable when the flight id parameter changed, but unfortunately that change doesn't get detected when passed down into the GoalFormComponent. So I'm deciding to leave this.

Reply:
Actually got this working... updating tests